client/src/layers/GroupManagement/index.tsx

```diff
@@ -1,67 +1,95 @@
-import React from 'react';
+import React, { useState } from 'react';
 import * as S from './style';
-import { DropDown } from '@/lib/design';
-import dots from '@public/icons/more_horiz.svg';
-import TeamManagementItem from '@/components/TeamManagementItem';
-import { useRecoilValue } from 'recoil';
-import userAtom from '@/recoil/user';
-import { useEffect } from 'react';
-import { useState } from 'react';
-import { getUsersByOrganization, UserProfile } from '@/lib/api/user';
+import { Modal } from '@/lib/design';
+
+import { deleteUserById, modifyUserAdminById } from '@/lib/api/user';
 import TeamInviteBar from '@/components/TeamInviteBar';
+import { TeamItemViewer } from '@/components/TeamItemViewer';
 
-const dropDownListForAdmin = [
-  {
-    id: 1,
-    name: '팀원에서 제외',
-  },
-  {
-    id: 2,
-    name: '팀원으로 변경',
-  },
-];
+const ADMIN = true;
+const TEAMMATE = false;
 
-const dropDownListForMember = [
-  {
-    id: 1,
-    name: '팀원에서 제외',
-  },
-  {
-    id: 2,
-    name: '관리자로 변경',
-  },
-];
+const generateMessage = (message: string) => {
+  return message === '팀원에서 제외' ? (
+    <S.ModalText>정말로 팀원에서 제외하겠습니까?</S.ModalText>
+  ) : message === '팀원으로 변경' ? (
+    <S.ModalText>정말 팀원으로 변경하시겠습니까?</S.ModalText>
+  ) : (
+    <S.ModalText>정말 관리자로 변경하시겠습니까?</S.ModalText>
+  );
+};
 
+// TODO: 함수 분리하기
 export const GroupManagement = () => {
-  const userState = useRecoilValue(userAtom);
-  const [userList, setUserList] = useState<Array<UserProfile>>([]);
-  useEffect(() => {
-    (async () => {
-      const newUserList = await getUsersByOrganization(userState.organization as number);
-      setUserList(newUserList);
-    })();
-  }, [userState.organization]);
+  const [showModal, setShowModal] = useState(false);
+  const [forceRender, setForceRender] = useState({ flag: false });
```

Comment:
그냥 이렇게 해도 되지 않나요?? 객체에 넣어주신 이유가 있나요?? 🤔

```suggestion
  const [forceRender, setForceRender] = useState(false);
```

Reply:
저도 같은 생각이에요. forceRender 가 어떤 의미인지도 궁금하네요! 내일 설명부탁드려요